package.json

```diff
@@ -4,6 +4,9 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
+    "build:api": "yarn workspace @tryrolljs/api  build",
```

Comment:
I'd not keep it in the root's package json. With the growth of monorepo, addition of the specified scripts will make it unreadable. It's easier to navigate into the package you want to build & run it there.